Plus last update leftovers + some vanilla templates additions

wersal454 opened the pull request:
## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:

leftover content from plus. All templates has been adopted to include new sniper patrol stuff
CASdive is still WIP
Changes to templates from plus      

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 

********************************************************
Notes:
WIP
CASdive needs testing
and sniper patrol stuff needs testing

wersal454 committed:
Update 3CBF_Riv_CHDKZ.sqf

wersal454 committed:
Update 3CBF_Riv_CHDKZ_CW.sqf

wersal454 committed:
Update 3CBF_Riv_TI.sqf

wersal454 committed:
Update Aegis_AI_AAF.sqf

wersal454 committed:
Update Aegis_AI_AFRF_Arid.sqf

wersal454 committed:
Update Aegis_AI_AFRF_Temperate.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Arid.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Temperate.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Tropical.sqf

wersal454 committed:
Update Aegis_AI_CSAT_Arid.sqf

wersal454 committed:
Update Aegis_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Aegis_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Aegis_AI_NATO_Arid.sqf

wersal454 committed:
Update Aegis_AI_NATO_Temperate.sqf

wersal454 committed:
Update Aegis_AI_NATO_Tropical.sqf

wersal454 committed:
Update RHS_AI_AFRF.sqf

wersal454 committed:
Update RHS_AI_CDF.sqf

wersal454 committed:
Update RHS_AI_USMC_Arid.sqf

wersal454 committed:
Update RHS_AI_USMC_Temperate.sqf

wersal454 committed:
Update RHS_AI_US_Army_Arid.sqf

wersal454 committed:
Update RHS_AI_US_Army_Temperate.sqf

wersal454 committed:
Update VN_AI_ARVN.sqf

wersal454 committed:
Update VN_AI_MACV.sqf

wersal454 committed:
Update VN_AI_PAVN.sqf

wersal454 committed:
Update VN_Riv_PL.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Tropical.sqf

wersal454 committed:
Update Vanilla_Reb_FIA.sqf

wersal454 committed:
Update Vanilla_Reb_LL.sqf

wersal454 committed:
Update Vanilla_Reb_SDK.sqf

wersal454 committed:
Update Vanilla_Reb_SDK.sqf

wersal454 committed:
Update Vanilla_Reb_Vehicle_Attributes.sqf

wersal454 committed:
Update WS_Riv_Exegermenos.sqf

wersal454 committed:
Update FactionExample.sqf

wersal454 committed:
Update 3CBF_AI_ARD.sqf

wersal454 committed:
Update BRAF_AI_BRAF_Tropical.sqf

wersal454 committed:
Update BWA3_AI_BW_Arid.sqf

wersal454 committed:
Update BWA3_AI_BW_Temperate.sqf

wersal454 committed:
Update CSA_AI_UK_Arctic.sqf

wersal454 committed:
Update CSA_AI_UK_Arid.sqf

wersal454 committed:
Update CSA_AI_UK_Temperate.sqf

wersal454 committed:
Update CSA_AI_WEH_Arctic.sqf

wersal454 committed:
Update CSA_AI_WEH_Arid.sqf

wersal454 committed:
Update CSA_AI_WEH_Temperate.sqf

wersal454 committed:
Update CUP_AI_ACR_Arid.sqf

wersal454 committed:
Update CUP_AI_ACR_Temperate.sqf

wersal454 committed:
Update CUP_AI_AFRF_Arctic.sqf

wersal454 committed:
Update CUP_AI_AFRF_Arid.sqf

wersal454 committed:
Update CUP_AI_AFRF_Arid.sqf

wersal454 committed:
Update CUP_AI_AFRF_Temperate.sqf

wersal454 committed:
Update CUP_AI_BAF_Arid.sqf

wersal454 committed:
Update CUP_AI_BAF_Temperate.sqf

wersal454 committed:
Update CUP_AI_BW_Arid.sqf

wersal454 committed:
Update CUP_AI_BW_Temperate.sqf

wersal454 committed:
Update CUP_AI_CDF_Arctic.sqf

wersal454 committed:
Update CUP_AI_CDF_Temperate.sqf

wersal454 committed:
Update CUP_AI_HIL.sqf

wersal454 committed:
Update CUP_AI_ION_Arctic.sqf

wersal454 committed:
Update CUP_AI_ION_Arid.sqf

wersal454 committed:
Update CUP_AI_RACS_Arid.sqf

wersal454 committed:
Update CUP_AI_RACS_Tropical.sqf

wersal454 committed:
Update CUP_AI_SLA_Temperate.sqf

wersal454 committed:
Update CUP_AI_TKA_Arid.sqf

wersal454 committed:
Update CUP_AI_US_Army_Arid.sqf

wersal454 committed:
Update CUP_AI_US_Marine_Arid.sqf

wersal454 committed:
Update CUP_AI_US_Marine_Temperate.sqf

wersal454 committed:
Update CW_AI_CIS.sqf

wersal454 committed:
Update CW_AI_CIS_WBK.sqf

wersal454 committed:
Update CW_AI_REP.sqf

wersal454 committed:
Update CW_AI_REP_WBK.sqf

wersal454 committed:
Update EMP_AI_EMP.sqf

wersal454 committed:
Update EMP_AI_EMP_WBK.sqf

wersal454 committed:
Update FFAA_AI_SPA_Arid.sqf

wersal454 committed:
Update FFAA_AI_SPA_Temperate.sqf

wersal454 committed:
Update GM_AI_BW.sqf

wersal454 committed:
Update GM_AI_BW_arctic.sqf

wersal454 committed:
Update GM_AI_NVA.sqf

wersal454 committed:
Update GM_AI_NVA_arctic.sqf

wersal454 committed:
Update IFA_AI_SOV_Arctic.sqf

wersal454 committed:
Update IFA_AI_SOV_Temperate.sqf

wersal454 committed:
Update IFA_AI_UK_Arctic.sqf

wersal454 committed:
Update IFA_AI_UK_Arid.sqf

wersal454 committed:
Update IFA_AI_UK_Temperate.sqf

wersal454 committed:
Update IFA_AI_USA_Arctic.sqf

wersal454 committed:
Update IFA_AI_USA_Arid.sqf

wersal454 committed:
Update IFA_AI_USA_Temperate.sqf

wersal454 committed:
Update IFA_AI_WEH_Arctic.sqf

wersal454 committed:
Update IFA_AI_WEH_Arid.sqf

wersal454 committed:
Update IFA_AI_WEH_Temperate.sqf

wersal454 committed:
Update NFtS_AI_SOV_Arctic.sqf

wersal454 committed:
Update NFtS_AI_SOV_Temperate.sqf

wersal454 committed:
Update OPTRE_AI_Covenant.sqf

wersal454 committed:
Update OPTRE_AI_Covenant_Grunts.sqf

wersal454 committed:
Update OPTRE_AI_Covenant_Grunts_New.sqf

wersal454 committed:
Update OPTRE_AI_Covenant_New.sqf

wersal454 committed:
Update OPTRE_AI_UNSC.sqf

wersal454 committed:
Update OPTRE_AI_UNSC_New.sqf

wersal454 committed:
Update OPTRE_AI_UNSC_desert.sqf

wersal454 committed:
Update OPTRE_AI_UNSC_desert_New.sqf

wersal454 committed:
Update OPTRE_AI_UNSC_snow.sqf

wersal454 committed:
Update OPTRE_AI_UNSC_snow_New.sqf

wersal454 committed:
Update OPTRE_AI_UNSC_tropical.sqf

wersal454 committed:
Update OPTRE_AI_UNSC_tropical_New.sqf

wersal454 committed:
Update PLA_AI_CHI_Arctic.sqf

wersal454 committed:
Update PLA_AI_CHI_Arid.sqf

wersal454 committed:
Update PLA_AI_CHI_Temperate.sqf

wersal454 committed:
Update PED_AI_ITA_Arid.sqf

wersal454 committed:
Update PED_AI_ITA_Temperate.sqf

wersal454 committed:
Update RACS_AI_Army_New.sqf

wersal454 committed:
Update RACS_AI_SLA_New.sqf

wersal454 committed:
Update SPE_IFA_AI_US.sqf

wersal454 committed:
Update SPE_IFA_AI_WEH.sqf

wersal454 committed:
Update STALKER_AI_ClearSky.sqf

wersal454 committed:
Update STALKER_AI_ClearSky_RHS.sqf

wersal454 committed:
Update STALKER_AI_Freedom.sqf

wersal454 committed:
Update STALKER_AI_Freedom_RHS.sqf

wersal454 committed:
Update STALKER_AI_Military.sqf

wersal454 committed:
Update STALKER_AI_Military_RHS.sqf

wersal454 committed:
Update STALKER_AI_Monolith.sqf

wersal454 committed:
Update STALKER_AI_Monolith_RHS.sqf

wersal454 committed:
Update SFP_AI_SWE_Arctic.sqf

wersal454 committed:
Update SFP_AI_SWE_Arid.sqf

wersal454 committed:
Update SFP_AI_SWE_Temperate.sqf

wersal454 committed:
Update Scion_AI_Empire.sqf

wersal454 committed:
Update Scion_AI_Republic.sqf

wersal454 committed:
Update TFC_AI_CA_Army_Arid.sqf

wersal454 committed:
Update TFC_AI_CA_Army_Temperate.sqf

wersal454 committed:
Update TMT_AI_Army.sqf

wersal454 committed:
Update UNS_AI_PAVN.sqf

wersal454 committed:
Update UNS_AI_US.sqf

wersal454 committed:
Update Stringtable.xml

wersal454 committed:
Update ForkChangelog.md

wersal454 committed:
Update README.md

wersal454 committed:
Update mapInfo.hpp

wersal454 committed:
Update description.ext

wersal454 committed:
Update description.ext

wersal454 committed:
Merge branch 'unstable' into plus-last-update-leftovers

wersal454 committed:
Merge branch 'unstable' into plus-last-update-leftovers

Silence-112 committed:
Merge branch 'unstable' into plus-last-update-leftovers

wersal454 committed:
Update Vanilla_Reb_Vehicle_Attributes.sqf

wersal454 committed:
Update cfgPlaneLoadouts.hpp

wersal454 committed:
Update fn_setPlaneLoadout.sqf

wersal454 committed:
Update Vanilla_Reb_Vehicle_Attributes.sqf

Silence-112 commented on A3A/addons/ultimate/config/plane/cfgPlaneLoadouts.hpp:
```diff
@@ -6,9 +6,20 @@ class planeLoadouts
     // missileLauncher[]
     // bombRacks[]
     // diveParams[]
+    class CASDIVE
+    {
+       class B_Plane_CAS_01_dynamicLoadout_F
+        {
+            loadout[] = {"","","","","PylonMissile_1Rnd_Bomb_04_F","PylonMissile_1Rnd_BombCluster_03_F","","","",""};
+            mainGun = "Gatling_30mm_Plane_CAS_01_F";
+            bombRacks[] = {"Bomb_03_Plane_CAS_02_F", "BombCluster_02_F"};
+            diveParams[] = {1200, 600, 180, 55, 15, {0,0}};
+        };
+    };
+
```
```suggestion
    class CASDIVE
    {
        class B_Plane_CAS_01_dynamicLoadout_F
        {
            loadout[] = {"","","","","PylonMissile_1Rnd_Bomb_04_F","PylonMissile_1Rnd_BombCluster_03_F","","","",""};
            mainGun = "Gatling_30mm_Plane_CAS_01_F";
            bombRacks[] = {"Bomb_04_Plane_CAS_01_F", "BombCluster_03_F"};
            diveParams[] = {1200, 600, 180, 55, 15, {0,0}};
        };
    };
```

wersal454 committed:
Update A3A/addons/ultimate/config/plane/cfgPlaneLoadouts.hpp

Co-authored-by: Silence <78276788+SilenceIsFatto@users.noreply.github.com>

wersal454 committed:
Update cfgPlaneLoadouts.hpp

wersal454 committed:
Update cfgPlaneLoadouts.hpp

wersal454 committed:
Update fn_setPlaneLoadout.sqf

wersal454 committed:
Update cup_veh_Civ.sqf

wersal454 committed:
hotfix

wersal454 committed:
Update fn_createAIcontrols.sqf

wersal454 committed:
Merge branch 'unstable' into plus-last-update-leftovers

wersal454 committed:
Update fn_setPlaneLoadout.sqf

wersal454 committed:
Update Aegis_AI_AAF.sqf

wersal454 committed:
Update Aegis_AI_AFRF_Arid.sqf

wersal454 committed:
Update Aegis_AI_AFRF_Temperate.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Arid.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Arid.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Temperate.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Arid.sqf

wersal454 committed:
Update Aegis_AI_AUKUS_Tropical.sqf

wersal454 committed:
Update Aegis_AI_CSAT_Arid.sqf

wersal454 committed:
Update Aegis_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Aegis_AI_NATO_Arid.sqf

wersal454 committed:
Update Aegis_AI_NATO_Temperate.sqf

wersal454 committed:
Update Aegis_AI_NATO_Tropical.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Tropical.sqf

wersal454 committed:
hotfix

wersal454 committed:
Update Aegis_AI_CSAT_Arid.sqf

wersal454 committed:
hotfix

wersal454 committed:
Update fn_setPlaneLoadout.sqf

wersal454 committed:
Update Aegis_AI_NATO_Temperate.sqf

wersal454 committed:
finished vanilla + aegis planes

wersal454 committed:
radar activation + flares upon finishing run

wersal454 committed:
radar and flares

wersal454 committed:
cas fix + additional planes + casdive full support

wersal454 committed:
typo fix

wersal454 committed:
TFC/AMR(+R3F) support/fix

wersal454 commented:
Ufff. This is gonna be long.

Changelog:
*Addonvehicles - added CIV planes(before they were in civ helis)
   updated civilian example with civ helis and planes, also added variants and animations
   Added sniper patrol pair for every template.(only templates that were in plus use cloaked gear)
   sniper patrol pair consist of patrolspotter and patrolsniper.
   In faction example added cloak version of gear(for snipers) - cloakUniforms,cloakVests,cloakRifles, etc.
*Some small changes to templates:
   additional vanilla helis to civ template,armored orca for CSAT 
   NATO's tracked APC is now IFV(because it's called an IFV lol)
   Vanilla and Aegis templates now use stealth variants of jets dlc planes as ASF
   Vanilla and Aegis templates now use jets dlc planes as CAS + F-35 in aegis used in CAS and in ASF(stealth variant)
   Now all vanilla rivals have helicopter
   Fixes and additions for vanilla rebels (rebels, now can by AAF tanks from tanks dlc + AAF small helicopter(unarmed)).
*CASdive: GIANT list of fixes for regular CAS plane loadouts
   support for CASdive for every plane.
  +Additional planes and their loadouts for clone wars, PLA, RACS+SLA,AMF,Pedagne
   CAS will now pop flares upon exiting it's gun run + it will use it's radar during the run(dunno if it will help AI or not).
   Vanilla and Aegis templates now use stealth variants of jets dlc planes as ASF
   Vanilla and Aegis templates now use jets dlc planes as CAS + F-35 in aegis used in CAS and in ASF(stealth variant).
*one small bug fix fn_createAIcontrols.
*Some small changes for VEHAIRDROP from plus (it will be used more... or less I dunno)
  Airdrop now uses groupsTierAirborne insted of groupsTierMedium.
  Stringtables (in eng and ru) for Sumava map.
*And some other changes from plus I probably forgot about.

stutpip123 commented:
"Vanilla and Aegis templates now use jets dlc planes as CAS + F-35 in aegis used in CAS and in ASF(stealth variant)
Now all vanilla rivals have helicopter"
DLC should only be used if activated in the setup menu for vanilla templates

"+Additional planes and their loadouts for clone wars, PLA, RACS+SLA,AMF,Pedagne"
Those clone wars planes were not added because they physically don't fit into hangars and therefore cannot be used in templates


hard to say if theres any issues without looking through all the code. its a lot of changes for one PR

Silence-112 commented:
Sounds like a perfect case for the unstable branch/test mod

stutpip123 commented:
not until the DLC thing is sorted out. Vanilla templates cannot REQUIRE DLC for anything. It's fine to use the system to check loaded DLC and append vehicles then, but cannot simply replace the vanilla fighter with DLC and have that as the only option

wersal454 commented:
> "Vanilla and Aegis templates now use jets dlc planes as CAS + F-35 in aegis used in CAS and in ASF(stealth variant) Now all vanilla rivals have helicopter" DLC should only be used if activated in the setup menu for vanilla templates
> 
> "+Additional planes and their loadouts for clone wars, PLA, RACS+SLA,AMF,Pedagne" Those clone wars planes were not added because they physically don't fit into hangars and therefore cannot be used in templates
> 
> hard to say if theres any issues without looking through all the code. its a lot of changes for one PR

1. I have not added the big ships/planes 2. Have you looked at plane from TFC? lol. It doesn't fit in vanilla hangar. It immediately blows up.

stutpip123 commented:
Well thats Silences fault. I didn't port TFC

wersal454 commented:
> not until the DLC thing is sorted out. Vanilla templates cannot REQUIRE DLC for anything. It's fine to use the system to check loaded DLC and append vehicles then, but cannot simply replace the vanilla fighter with DLC and have that as the only option

Yeah probably something should be done about that, but that is how vanilla templates always were so ¯\_(ツ)_/¯

stutpip123 commented:
https://github.com/SilenceIsFatto/A3-Antistasi-Ultimate/blob/66b6997ddd24aae97e674a90a80332d267289661/A3A/addons/core/Templates/Templates/BWA3/BWA3_AI_BW_Temperate.sqf
See the BWMod templates for how to implement optional content to arrays such as for PlanesAA

Silence-112 commented:
> Well thats Silences fault. I didn't port TFC

We have used TFC multiple times, surprised we didn't figure it out sooner

wersal454 committed:
dlcs checks + fixes

wersal454 committed:
small fixes

wersal454 committed:
fix №2.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Temperate.sqf

stutpip123 commented on A3A/addons/core/Templates/Templates/Vanilla/Vanilla_AI_NATO_Temperate.sqf:
```diff
@@ -35,22 +37,22 @@ private _armedVehicles = ["B_T_MRAP_01_gmg_F", "B_T_MRAP_01_hmg_F"];
-["vehiclesPlanesAA", ["B_Plane_Fighter_01_F"]] call _fnc_saveToTemplate;
-["vehiclesPlanesTransport", ["B_VTOL_01_infantry_F"]] call _fnc_saveToTemplate;
+private _planeCAS = ["B_Plane_CAS_01_dynamicLoadout_F"];
+private _planeAA = [];
```
does this mean if Jets is not loaded then they dont get an AA plane? what happened to the pure vanilla plane?

wersal454 replied:
"B_Plane_Fighter_01_F" never had a AA loadout, can add it tho

wersal454 committed:
Update Vanilla_AI_NATO_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Tropical.sqf

wersal454 committed:
Update Vanilla_Civ.sqf

wersal454 committed:
Update fn_setPlaneLoadout.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Tropical.sqf

wersal454 committed:
Update Vanilla_Reb_FIA.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Arid.sqf

wersal454 committed:
Update Vanilla_AI_CSAT_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_LDF.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Arid.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Temperate.sqf

wersal454 committed:
Update Vanilla_AI_NATO_Tropical.sqf

Silence-112 commented:
I'll merge this if you're fine with it @wersal454

wersal454 commented:
> I'll merge this if you're fine with it @wersal454

hold up, finishing touches

wersal454 committed:
Update Vanilla_Reb_FIA.sqf

wersal454 committed:
Update Vanilla_Reb_FIA.sqf

wersal454 committed:
Update Vanilla_Reb_FIA.sqf

wersal454 committed:
Update Vanilla_Reb_LL.sqf

wersal454 committed:
Update Vanilla_Reb_SDK.sqf

wersal454 committed:
Update Vanilla_Reb_LL.sqf

wersal454 committed:
Update Vanilla_Reb_SDK.sqf

wersal454 committed:
Update Vanilla_Reb_FIA.sqf

wersal454 committed:
Update Vanilla_AI_AAF.sqf

wersal454 committed:
Update Vanilla_Reb_FIA.sqf

wersal454 committed:
Update Vanilla_Reb_LL.sqf

wersal454 committed:
Update Vanilla_Reb_SDK.sqf

Silence-112 commented:
Is this good yet or no?

wersal454 commented:
> Is this good yet or no?

no, finishing vanilla rivals

Silence-112 commented:
np, just comment when it is

wersal454 committed:
Update Vanilla_Riv_Exegermenos.sqf

wersal454 commented:
Changelog №2:
AAF:
AAF will now use only only theirs 230mm artillery
AAF tracked APC now is IFV
CSAT:
CSAT's tracked APC is also moved to IFV category
no transport plane without APEX dlc enabled
some fixes to helicopters categorisation
probably mentioned that already but CSAT will now use ION's armored orca with CSAT camo
CSAT will now use AAF's 230mm artillery with CSAT camo
CSAT will now have railgun tank from the latest update
LDF:
basicly same as above.
NATO:
Added mover variant of the truck to basic vehicles, since it now has 1 cargo slot
Added AAF MRAP with NATO camo variant
Same template changes as above + If apex dlc is enabled transport helis will now have CTRG variant of vanilla NATO transport helicopter
Mortar APC from WS was added to the artillery
Civ:
Many dlc checks were added to vehicles + some missing vehicles were added
Civis will now wear costumes from Contact and Arts of war dlcs
Reb:
Many dlc checks were added to vehicles
by default FIA & LL rebels will now don't have planes until APEX is enabled (may break some shit, same as missing looting heli)
Now rebels can by civ versions of  AAF heli and CSAT heli
some missing vehicles were added
WS armored offroads were moved from blackmarket to regular store
On the blackmarket now you will be able to by: 
small uavs from dlc's (for example deming UGV + IDAPs antimine uav)
Static designators from marksmen dlc
automatic HMG and GMG
Armed and unarmed vanilla UGVs
AAF's light tanks from tanks dlc
3 new helicopters (2 armed ones)
Tons of new clothing + Art of war dlc backpacks
new heads
camera as a starting equipment(binocular) if you have WS.
Riv:
checks for dlc were redone.
now if you have APEX dlc enabled Exegermenos will now have both vanilla offroad and APEX one
uav's from Laws of war and WS were added to rivals arsenal
If you have WS enabled rivals will have light CSAT APC from WS(armed and unarmed)
If you have Tanks dlc enabled rivals will now have autocannon wiesel from the dlc
Rivals will now use more gear(not weapons) with and without dlc enabled

Special forces units will now sometimes wear CBRN suits if you enable the Contact dlc
No new weapons from dlc were added (no time) something for the future update
Tons of camo variants
probably mentioned that already, artillery can now fire cluster and mines munition (if I understand that correctly) or they will now fire 32 rounds of HE ammo and then switch to cluster and etc.
Enemy templates will now use vanilla planes as AA without JETS dlc enabled 
Tons of checks for dlc's(for example no light tanks until you enable the dlc)
Templates will now use police vans from Laws of war dlc
No more radar and sam until you enable the Jets dlc
Many additions to camo variants of the enemy vehicles
Officers will now wear officer uniforms from Art of war dlc if you enable that dlc
Sometimes you will spot enemies wearing gasmask from contact dlc
Enemy squad leaders will now have IR grenades in their inventory
Elite squads can now spawn with shield backpack from WS cdls (no clue if they will use it or not)

wersal454 committed:
Merge branch 'unstable' into plus-last-update-leftovers

wersal454 committed:
Merge branch 'unstable' into plus-last-update-leftovers

Silence-112 commented:
That is a chonky changelog indeed
Good to go yet?

wersal454 commented:
yes
